db/seeds.ts

```diff
@@ -50,6 +50,7 @@ const seed = async () => {
     { wikidata: "Q429785", name: "Poster", schema: "Poster" },
     { wikidata: "Q59094171", name: "Ethical review", schema: "CreativeWork" },
     { wikidata: "Q580922", name: "Preprint", schema: "CreativeWork" },
+    { wikidata: "Q1744627", name: "Classification Algorithm", schema: "CreativeWork" },
```

Comment:
I noticed that existing names use sentence case (e.g., "Ethical review") while this change uses the title case. Is this intended? 

In case we want to keep title case:

```suggestion
    { wikidata: "Q1744627", name: "Classification algorithm", schema: "CreativeWork" },
```